docs/concepts/cluster-topologies.md

```diff
@@ -28,4 +28,4 @@ Ondat performs file Input/Output (I/O) operations over the network, which is how
 - The cluster topology can also aid in avoiding downtime issues that can arise from unaccounted resource/capacity planning and allocation for workloads, since storage-optimised nodes and compute-optimised workloads are compartmentalised.
 - In addition, another suitable use case for this topology is for elastic worker node fleets with burst-able workloads. A fleet can be quickly expanded with new worker nodes for compute-intensive workloads on demand, whilst maintaining a centralised data storage pool that is not impacted by rapid auto cluster scaling.
 - To configure this cluster topology for a new Ondat deployment, cluster administrators would need to apply an Ondat node label called `storageos.com/computeonly` to nodes, which would inform Ondat that it *should not* use the nodes to join a storage pool.
-  - Review the [Feature Labels](https://docs.ondat.io/docs/reference/labels/) reference page for more information on how to enable Ondat features correctly.
+  - Review the [Centralised Cluster Topology](/docs/reference/compute-only/) operations page for more information on how use this topology model for your clusters.
```

Comment:
information on how **to** use this topology

Reply:
fixed!